tasks/wp-rev.js

```diff
@@ -35,12 +36,19 @@ module.exports = function (grunt) {
                 var hash = md5(f, options.algorithm, 'hex'),
                 prefix = hash.slice(0, options.length),
                 filename = path.basename(f),
-                renamed = [prefix, filename].join('.'),
-                outPath = path.resolve(path.dirname(f), renamed),
-                regex = new RegExp('([0-9a-f]*\\.)*?' + filename),
+                ext = path.extname(filename),
+                baseName = path.basename(filename, ext),
+                renamed;
+                if ( options.revert ) {
```

Comment:
Please remove the spaces before and after options.revert. After that I'll merge it in =)